pendulum_msgs/msg/JointState.msg

```diff
@@ -0,0 +1,3 @@
+float64 position
```

Comment:
Since the message size is unknown based on the definition I don't think we can make the complete pipeline preallocated here. That is the reason why you can specify upper bounds now in the message definition. Only then we can ensure to preallocate data objects throughout the pipeline.